src/lib/y2storage/disk_size.rb

```diff
@@ -609,10 +609,48 @@ def floor(unit_size)
     # @example
     #   x = DiskSize.KB(1)   #=> <DiskSize 0.98 KiB (1000)>
     #   x.to_human_string    #=> "0.98 KiB"
-    def to_human_string
+    def to_human_string(round_method = :round)
```

Comment:
Moreover, for those optional methods that are kind of unexpected[*] and have a default value, I usually prefer keywords attributes, like
```ruby
def to_human_string(round_method: :round)
```
In this same class, there are similar examples of such arguments, like `rounding` or `legacy_units`. So consistency would be another argument here.

But feel free to disagree, of course.

[*] Unexpected because I would usually not expect a `to_xx` to receive an argument and also because they are not really needed to perform the integral part of the functionality, they are there mostly to influence the functionality with some fancy hue.